evidence/andresilva/0001-andresilva-2024-H1-evidence.md

```diff
@@ -0,0 +1,37 @@
+# Evidence-0001: Retention at Rank 5
+
+|                  |             |
+|------------------|-------------|
+| **Report Date**  | 2024/07/31  |
+| **Submitted by** | André Silva |
+
+## Member details
+
+- Matrix username: `@andresilva:matrix.org`
+- Polkadot address: `14YDyDZ9o1Nr2hMqLSbjYpr4Wm5s1gux6CvjYZfUTJ4Np3w1`
+- Current rank: `5`
+- Date of initial induction: `2024/02/05`
```

Comment:
Since andre was inducted on 24th November 2022, the date of initial induction should read "seeding".